test/web-platform-tests/to-run.yaml

```diff
@@ -202,9 +202,9 @@ CharacterData-appendChild.html: [fail, Unknown]
 Comment-constructor.html: [fail, Unknown]
 DOMImplementation-createDocument.html: [fail, Unknown]
 Document-URL.sub.html: [fail, Unknown]
-Document-characterSet-normalization.html: [fail, Unknown]
+Document-characterSet-normalization.html: [fail, Some encodings are not supported - see the whatwg-encoding module]
```

Comment:
Hmm, so is it a WPT bug that it's using encodings that are not in the Encoding Standard?

Reply:
They are in the Encoding Standard, but `iconv-lite` which is used by `whatwg-encoding` [does not support](https://github.com/jsdom/whatwg-encoding#unsupported-encodings) all of them yet.